pkg/meta/meta.go

```diff
@@ -885,6 +917,9 @@ func (m *Meta) DropTableOrView(dbID int64, tblID int64) error {
 	if err := m.txn.HDel(dbKey, tableKey); err != nil {
 		return errors.Trace(err)
 	}
+	if m.needUpdateName && len(dbName) > 0 && len(tbName) > 0 {
```

Comment:
Could we remove `len(dbName) > 0 && len(tbName) > 0`? In this function, it seems `dbName` and `tbName` will not be nil.